helix-core/src/test/java/org/apache/helix/integration/manager/ClusterManager.java

```diff
@@ -34,7 +34,7 @@
 
 public class ClusterManager extends ZKHelixManager implements Runnable, ZkTestManager {
   private static Logger LOG = LoggerFactory.getLogger(ClusterControllerManager.class);
-  private static final int DISCONNECT_WAIT_TIME_MS = 3000;
+  private static final int DISCONNECT_WAIT_TIME_MS = 300;
```

Comment:
This value is used by all the participants (using MockParticipantManager) to finish off/disconnect when the cluster teardown is happening. Reducing this so that current thread is released faster.